Composer/packages/client/src/pages/botProject/create-publish-profile/PublishProfileDialog.tsx

```diff
@@ -131,6 +124,16 @@ export const PublishProfileDialog: React.FC<PublishProfileDialogProps> = (props)
     }
   }, [current, projectId]);
 
+  function formatDialogTitle() {
```

Comment:
Can you change this to arrow function? Also, we should move this out of the scope of the component and pass `current` as a parameter.

Reply:
done